content/development/how-to-write-a-loader.md

```diff
@@ -17,6 +19,28 @@ The loader is expected to give back one or two values. The first value is a resu
 
 In the complex case, when multiple loaders are chained, only the last loader gets the resource file and only the first loader is expected to give back one or two values (JavaScript and SourceMap). Values that any other loader give back are passed to the previous loader.
 
+In other words the loading order for chained loaders goes from right to left or bottom to top. 
+For example: lets say you have two loaders that go by the name of `fooLoader` and `barLoader`. You would like to execute `fooLoader` and then pass the result of the transformation from `fooLoader` finally to `barLoader`. 
```

Comment:
You should also probably use `foo-loader` / `bar-loader` vs. camel case seeing as that's how most loaders are packaged/named.